README.md

```diff
@@ -87,7 +87,28 @@ The most important changes are:
   sequence to differentiate conflicting slug, but rather a UUID (e.g. something
   like `2bc08962-b3dd-4f29-b2e6-244710c86106`). This makes the
   codebase simpler and more reliable when running concurrently, at the expense
-  of uglier ids being generated when there are conflicts.
+  of uglier ids being generated when there are conflicts. 
+* If you need previously used numeric sequence for some reason:
```

Comment:
I think we should remove phrases like `for some reason` as this potentially alienates those with this requirement by making their use case seem trivial or nonsensical.  Perhaps I'm reading too far into it, but if I do then perhaps others will too. :blue_heart: